### PR TITLE
Create eLife_Application_2019

### PR DESCRIPTION
**Pull request by GeorgiaHCA**
Putting the eLife application into markdown (new branch)

### CODE REVIEW

File: Georgia-Aitkenhead/eLife_Application_2019 [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# eLife Application
+### *Application for [eLife Innovation Sprint 2019](https://sprint.elifesciences.org)*
+
+## At the Innovation Sprint, I would like to work on... 
+### Please tell us briefly which problem(s) you would like to address in relation to research communication, open science, and/or applying the latest technologies to research and discovery (100 words)
```
Comment (KirstieJane) [inline Georgia-Aitkenhead/eLife_Application_2019]:
```suggestion
*Please tell us briefly which problem(s) you would like to address in relation to research communication, open science, and/or applying the latest technologies to research and discovery (100 words)*
```

File: Georgia-Aitkenhead/eLife_Application_2019 [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# eLife Application
+### *Application for [eLife Innovation Sprint 2019](https://sprint.elifesciences.org)*
```
Comment (KirstieJane) [inline Georgia-Aitkenhead/eLife_Application_2019]:
```suggestion
***Application for [eLife Innovation Sprint 2019](https://sprint.elifesciences.org)***
```

File: Georgia-Aitkenhead/eLife_Application_2019 [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+I previously worked on the government’s data, digital and technology fast stream, where I developed technical, policy, and project management skills rotating across multiple government departments, including: 
+  * HMRC’s behavioural insights team, making policy recommendations based on researching behaviours/technology. Developed skills in A/B testing, user research, and behavioural insights. 
+  * The Foreign Office, helping oversee GDPR compliance and supporting open data and transparency. Developed skills in data cleansing, data mapping, and data policy.    
+  * Government Digital Services: working on an agile team developing data analysis tools for gov.uk. Developed skills in stakeholder engagement and agile development.   
+
+I have also worked in occupational psychology facilitating change in the Scottish NHS. 
+
+## I am interested in participating in the Innovation Sprint because… 
+### Please tell us briefly what you hope to gain from participating in the event.(100 words)
```
Comment (KirstieJane) [inline Georgia-Aitkenhead/eLife_Application_2019]:
```suggestion
*Please tell us briefly what you hope to gain from participating in the event (100 words)* 
```

File: Georgia-Aitkenhead/eLife_Application_2019 [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+## I am interested in participating in the Innovation Sprint because… 
+### Please tell us briefly what you hope to gain from participating in the event.(100 words) 
+
+I hope to encounter new concepts, technologies and ideas, sparking extra possibilities and solutions for the research project I’m working on, and beyond. 
+I am interested in learning from the varied perspective of researchers, designers and developers. 
+I would like to contribute to discussions and strategies for embedding more open and transparent ways of conducting scientific research, and to share learning as well as gain additional insight into ways of putting new methodologies into practice. 
+The sprint is also a great opportunity to create ongoing connections with people from different fields and areas of expertise for future collaboration. 
+
+## Tell us about something you have created that you are proud of:
+### Please share the story of an idea you have prototyped or brought to fruition before, what you contributed and learned, and why you are proud of this.
```
Comment (KirstieJane) [inline Georgia-Aitkenhead/eLife_Application_2019]:
```suggestion
*Please share the story of an idea you have prototyped or brought to fruition before, what you contributed and learned, and why you are proud of this.*
```

File: Georgia-Aitkenhead/eLife_Application_2019 [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+### Please tell us briefly what you hope to gain from participating in the event.(100 words) 
+
+I hope to encounter new concepts, technologies and ideas, sparking extra possibilities and solutions for the research project I’m working on, and beyond. 
+I am interested in learning from the varied perspective of researchers, designers and developers. 
+I would like to contribute to discussions and strategies for embedding more open and transparent ways of conducting scientific research, and to share learning as well as gain additional insight into ways of putting new methodologies into practice. 
+The sprint is also a great opportunity to create ongoing connections with people from different fields and areas of expertise for future collaboration. 
+
+## Tell us about something you have created that you are proud of:
+### Please share the story of an idea you have prototyped or brought to fruition before, what you contributed and learned, and why you are proud of this. 
+The idea does not need to relate to the scope of this event, but we are particularly interested in contributions to any open (source or science) projects! (100 words)
```
Comment (KirstieJane) [inline Georgia-Aitkenhead/eLife_Application_2019]:
```suggestion
*The idea does not need to relate to the scope of this event, but we are particularly interested in contributions to any open (source or science) projects! (100 words)*
```

File: Georgia-Aitkenhead/eLife_Application_2019 [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+The idea does not need to relate to the scope of this event, but we are particularly interested in contributions to any open (source or science) projects! (100 words) 
+
+I worked with the Foreign Office’s historians to openly publish their historical diplomatic list by organising and running a hackathon. 
+I brought together experts from industry, NGOs, government and academia, and enlisted skilled developers and data scientists. 
+I am proud of the event because it created links between government and other sectors, and furthered the Foreign Office’s value of transparency. 
+It achieved interesting data visualisations and interfaces, and a cleansed dataset enriched with additional data sources: a valuable research resource which can be used to investigate topics such as changing demographics of diplomats and how postings correlate to historical events. 
+
+## Have you been or are you currently involved with eLife? 
+### Please list any associations with eLife, such as being a participant at last year's Sprint, an eLife Ambassador or ECAG member, involvement in a project or initiative supported by eLife, or having worked directly with us before.
```
Comment (KirstieJane) [inline Georgia-Aitkenhead/eLife_Application_2019]:
```suggestion
*Please list any associations with eLife, such as being a participant at last year's Sprint, an eLife Ambassador or ECAG member, involvement in a project or initiative supported by eLife, or having worked directly with us before.*
```

File: Georgia-Aitkenhead/eLife_Application_2019 [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+I worked with the Foreign Office’s historians to openly publish their historical diplomatic list by organising and running a hackathon. 
+I brought together experts from industry, NGOs, government and academia, and enlisted skilled developers and data scientists. 
+I am proud of the event because it created links between government and other sectors, and furthered the Foreign Office’s value of transparency. 
+It achieved interesting data visualisations and interfaces, and a cleansed dataset enriched with additional data sources: a valuable research resource which can be used to investigate topics such as changing demographics of diplomats and how postings correlate to historical events. 
+
+## Have you been or are you currently involved with eLife? 
+### Please list any associations with eLife, such as being a participant at last year's Sprint, an eLife Ambassador or ECAG member, involvement in a project or initiative supported by eLife, or having worked directly with us before. 
+We may take this into account when selecting between highly rated applications; other factors are of higher priority.
```
Comment (KirstieJane) [inline Georgia-Aitkenhead/eLife_Application_2019]:
```suggestion
*We may take this into account when selecting between highly rated applications; other factors are of higher priority.*
```

**Comment by KirstieJane** [comment]
Pinging @GeorgiaHCA to address the comments above and then merge.

**Comment by GeorgiaHCA** [comment]
Awesome! Thanks for looking over it :)

**Pull request merged**